### ISSUE TITLE
Soul Reaper doing incorrect damage.

### ISSUE THREAD

**Issue by wootlekins** [dalaranwow/dalaran-wow#5829]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Soul Reaper does not do the correct amount of damage on hit or during tick.

**Expected behaviour**: Sould Reaper should hit for 50% weapon damage from Lich King and tick for 70,000 shadow damage for 5 seconds (25m Heroic)

**Steps to reproduce the problem**:

1. Start Lich King 25m Heroic encounter (be a tank).
2. Let Lich King hit you with Soul Reaper during phase 2
3. Notice how much the initial hit damages for and the subsequent ticks hit for.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Screenshot of World of Logs damage for Soul Reaper as it currently is: https://imgur.com/a/xVdhI6d

Screenshot of how Soul Reaper should actually work: https://imgur.com/a/oMCPpr9

With the amount of teams running ICC25m Heroic now this should be fixed to keep the encounter challenging.

Thanks

**Comment by mrnuggets** [comment]
Hello devs?

**Comment by wootlekins** [comment]
Shitty report and devs don't care.

**Comment by mrnuggets** [comment]
Devs never cared. KEKW
